sdk/go/postgresql/defaultPrivileg.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 //
-// Deprecated: postgresql.DefaultPrivileg has been deprecated in favour of postgresql.DefaultPrivileges
+// Deprecated: postgresql.index.DefaultPrivileg has been deprecated in favour of postgresql.index.DefaultPrivileges
```

Comment:
Hmm, probably this `index` is not a correct module

Reply:
Ohhhh this is a good one! There are no subpackages in this repo - what should we do here?

Reply:
Probably keep the old version (`postgres.DefaultProvileg`)